app/Traits/GlobalScopes.php

```diff
@@ -1,41 +0,0 @@
-<?php
-
-namespace App\Traits;
-
-use Carbon\Carbon;
-use DB;
-
-trait GlobalScopes
-{
-    /**
-     * From the start of last month till now.
-     * @author Christopher Kelker 2020-02-18
-     * @return $query
-     */
-    public function scopeCreatedWithinLastMonth($query)
```

Comment:
This no longer needed?

Reply:
I couldn't find any evidence that it was.  But double-check.